services/server/deps.go

```diff
@@ -69,3 +70,9 @@ type supervisordService interface {
 type telemetryService interface {
 	DistributionMethod() serverpb.DistributionMethod
 }
+
+// checksService is a subset of methods of checks.Service used by this package.
+// We use it instead of real type for testing and to avoid dependency cycle.
+type checksService interface {
```

Comment:
Please add `//go:generate mockery` comment on top of the file.

Reply:
Done